exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -49,6 +49,7 @@
   String USER_RPC_TIMEOUT = "drill.exec.rpc.user.timeout";
   String METRICS_CONTEXT_NAME = "drill.exec.metrics.context";
   String USE_IP_ADDRESS = "drill.exec.rpc.use.ip";
+  String BIT_ADVERTISED_HOST = "drill.exec.rpc.bit.advertised.host";
```

Comment:
@paul-rogers Thank you for your helpful suggestions.  I understand what you are saying, and indeed, I was using exactly the flow you outlined above for my own use case.

Let me make sure I understand what you are proposing.  We can either:

1. Access the host name as an environment variable, in which case we could access it via `System.getEnv(...)`
2. Pass the host name in as a system variable (``-Ddrill.exec.rpc.bit.advertised-host=myhost`), which could just override the parameter I've already specified in `ExecConstants.java`

In either case, I guess I don't understand why I'd need to add anything to `drill-env.sh`, unless you mean just to add a commented out line for documentation purposes.  In the second case, is there any change I would need to make at all?